docs/en/administration/management/BE_configuration.md

```diff
@@ -2794,38 +2794,32 @@ When this value is set to less than `0`, the system uses the product of its abso
 - Introduced in: -
 -->
 
-<!--
 ##### parquet_late_materialization_enable
 
 - Default: true
 - Type: Boolean
 - Unit: -
 - Is mutable: No
-- Description:
+- Description: A boolean value to control whether to enable the late materialization of Parquet reader to improve performance. `true` indicates enabling late materialization, and `false` indicates disabling it.
 - Introduced in: -
--->
 
-<!--
 ##### parquet_late_materialization_v2_enable
 
 - Default: true
 - Type: Boolean
 - Unit: -
 - Is mutable: No
-- Description:
-- Introduced in: -
--->
+- Description: A boolean value to control whether to enable the late materialization v2 of Parquet reader to improve performance. `true` indicates enabling late materialization v2, and `false` indicates disabling it. In v3.3, only new version late materialization is reserved, and this configuration is deprecated.
```

Comment:
I don't understand. Is this one v2 only?

Reply:
update the wording. v2 is deprecated in 3.3, only `parquet_late_materialization_enable` is used.